docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx

```diff
@@ -0,0 +1,281 @@
+
+Here's an example:
+
+```html
+<video width="320" height="240" controls preload="none">
+  <source src="/path/to/video.mp4" type="video/mp4" />
+  <track
+    src="/path/to/captions.vtt"
+    kind="subtitles"
+    srclang="en"
+    label="English"
+  />
+  Your browser does not support the video tag.
+</video>
+```
+
+### Common `<video>` Tag Attributes
+
+| Attribute  | Description                                                                                               | Example Value                              |
+| ---------- | --------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| `src`      | Specifies the source of the video file.                                                                   | `<video src="/path/to/video.mp4"></video>` |
+| `width`    | Sets the width of the video player.                                                                       | `<video width="320"></video>`              |
+| `height`   | Sets the height of the video player.                                                                      | `<video height="240"></video>`             |
+| `controls` | If present, it displays the default set of playback controls.                                             | `<video controls></video>`                 |
+| `autoplay` | Automatically starts playing the video when the page loads. Note: Autoplay policies vary across browsers. | `<video autoplay></video>`                 |
+| `loop`     | Loops the video playback.                                                                                 | `<video loop></video>`                     |
+| `muted`    | Mutes the audio by default. Often used with `autoplay`.                                                   | `<video muted></video>`                    |
+| `preload`  | Specifies how the video is preloaded. Values: `none`, `metadata`, `auto`.                                 | `<video preload="none"></video>`           |
+
+For a comprehensive list of video attributes, refer to the MDN documentation [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attributes).
+
+### Accessibility Considerations
+
+- **Fallback Content:** Provide alternative content for browsers that do not support the `<video>` tag.
+- **Subtitles or Captions:** Include subtitles or captions for users who are deaf or hard of hearing. Utilize the `<track>` tag with your `<video>` elements to specify caption file sources.
+- **Accessible Controls:** Ensure video controls are navigable via keyboard and compatible with screen readers. This can be achieved by using standard HTML5 video controls or by creating custom controls that comply with ARIA (Accessible Rich Internet Applications) standards.
```

Comment:
```suggestion
- **Accessible Controls:** Ensure navigation with video controls works with keyboard and compatible with screen readers. This can be achieved by using standard HTML5 video controls or by creating custom controls that comply with ARIA (Accessible Rich Internet Applications) standards.
```